src/ucp/core/ucp_ep.c

```diff
@@ -3404,3 +3404,17 @@ ucs_status_t ucp_ep_query(ucp_ep_h ep, ucp_ep_attr_t *attr)
 
     return UCS_OK;
 }
+
+unsigned ucp_ep_rkey_pack_uct_flags(ucp_ep_h ep)
+{
+    unsigned flags = 0;
+
+    if ((ucp_ep_config(ep)->key.err_mode ==
+         UCP_ERR_HANDLING_MODE_PEER) &&
+        /* and at least 1 lane is connect2iface */
+        (ucp_ep_config(ep)->p2p_lanes == 0)) {
```

Comment:
how this condition makes sure that at least 1 lane is connect2iface?

Reply:
((num_lanes() - !!has_cm_lane()) != pop_count(ucp_ep_config(ep)->p2p_lanes))
or
ucp_ep_config(ep)->p2p_lanes & (UCS_MASK(num_lanes()) & ~UCS_BIT(get_cm_lane())